gradle/java.gradle

```diff
@@ -98,3 +104,7 @@ plugins.withId('java') {
         testCompile "org.mockito:mockito-core:${mockitoVersion}"
     }
 }
+
+def getDefaultJavaVersion() {
```

Comment:
I see you defined this in 2 places. Since this file is imported into main `build.gradle` file, does it suffice to just declare it here?

Reply:
We can pull it out of `java.gradle` but not `build.gradle`. Will remove this declaration.